com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/jobs/AbortedOperationsCleaner.java

```diff
@@ -40,11 +41,11 @@ public AbortedOperationsCleaner(HistoricOperationEventService historicOperationE
 
     @Override
     public void execute(Date expirationTime) {
-        Calendar now = Calendar.getInstance();
-        now.add(Calendar.MINUTE, -30);
+        Instant instant = Instant.now()
+                                 .minus(30, ChronoUnit.MINUTES);
```

Comment:
Is this bring some improvement?

Reply:
Usage of the new Java 8 Time API is in general recommended over the old `Date` and `Calendar` APIs.